tests/e2e/util.go

```diff
@@ -316,6 +320,75 @@ func createPVC(client clientset.Interface, pvcnamespace string, pvclaimlabels ma
 	return pvclaim, err
 }
 
+// createPVC helps creates pvc with given namespace and labels using given storage class
+func scaleCreatePVC(client clientset.Interface, pvcnamespace string, pvclaimlabels map[string]string, ds string, storageclass *storagev1.StorageClass, accessMode v1.PersistentVolumeAccessMode, wg *sync.WaitGroup) {
+	worker := GetAndExpectIntEnvVar(envWorkerPerRoutine)
+	defer wg.Done()
+	if worker == 1 {
+		pvcspec := getPersistentVolumeClaimSpecWithStorageClass(pvcnamespace, ds, storageclass, pvclaimlabels, accessMode)
+		ginkgo.By(fmt.Sprintf("Creating PVC using the Storage Class %s with disk size %s and labels: %+v accessMode: %+v", storageclass.Name, ds, pvclaimlabels, accessMode))
+		pvclaim, err := fpv.CreatePVC(client, pvcnamespace, pvcspec)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred(), fmt.Sprintf("Failed to create pvc with err: %v", err))
+		pvclaims = append(pvclaims, pvclaim)
+	}
+	for index := 1; index <= worker/2; index++ {
+		pvcspec := getPersistentVolumeClaimSpecWithStorageClass(pvcnamespace, ds, storageclass, pvclaimlabels, accessMode)
+		ginkgo.By(fmt.Sprintf("Creating PVC using the Storage Class %s with disk size %s and labels: %+v accessMode: %+v", storageclass.Name, ds, pvclaimlabels, accessMode))
+		pvclaim, err := fpv.CreatePVC(client, pvcnamespace, pvcspec)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred(), fmt.Sprintf("Failed to create pvc with err: %v", err))
+		pvclaims = append(pvclaims, pvclaim)
+	}
+}
+
+// scaleCreateDeletePVC helps create and delete pvc with given namespace and labels using given storage class
+func scaleCreateDeletePVC(client clientset.Interface, pvcnamespace string, pvclaimlabels map[string]string, ds string, storageclass *storagev1.StorageClass, accessMode v1.PersistentVolumeAccessMode, wg *sync.WaitGroup, lock *sync.Mutex) {
```

Comment:
what is this method doing, please add more comments

Reply:
Added the same